core/embed/rust/src/micropython/iter.rs

```diff
@@ -21,15 +21,14 @@ impl IterBuf {
     }
 }
 
-pub struct Iter<'a> {
```

Comment:
> Perhaps iter_buf should be transformed into PhantomData<'a IterBuf> instead of removing?

won't help because we actually need to mutably grab that particular IterBuf instance